bndtools.core/resources/unprocessed/serviceTemplates/java-service-impl/bnd.bnd

```diff
@@ -0,0 +1,12 @@
+-buildpath: \
+	osgi.annotation,\
+	org.osgi.service.component.annotations,\
+	$api_package$
+
+$if(javaLevel)$
```

Comment:
same, handle on workspace level

Reply:
Isn't the javaLevel set during project creation?   In that case shouldn't it be project level (as would be expected by wizard user)?

Reply:
We tend to set the Java level once in the workspace and not change it in the projects. This makes it a lot easier to manage, and since the projects have a shared release cycle it is not a big deal.

Reply:
Yes, but the project creation wizards (even the bnd project creation wizards) explicitly have javalevel setting for that project on the first page (i.e. it's almost the first thing they see when creating a new project).   I think it would be terribly shocking to have that fail to work as expected for someone that's used to having it there for 100 years or so. :).   I suggest that if you don't want to surprise people that have been using the pre-bnd way wrt workspaces for a long time...remove the javaLevel setting from at least the bnd project wizards (you can't of course remove them from the Java project wizards).